tests/behat/features/big_pipe.feature

```diff
@@ -12,3 +12,28 @@ Feature: Check that BigPipeTrait works for or D9
     Given I install a "big_pipe" module
     When I visit "/"
     Then cookie "big_pipe_nojs" does not exist
+
+  @api
+  Scenario: Assert that Big Pipe cookie is set when user logs into multiple users in a scenario
```

Comment:
@richardgaunt 
the description talks about logging in "into multiple users". I'm not sure what this means.

If this means - log in AS multiple users - I do not see multiple logins in the test.

Is the purpose of these tests PR to demonstrate that a cookie "setting" is preserved during multiple login sessions within a single scenario?

Reply:
First user is anonymous, second user is logged in - 2 users

Reply:
Yes the purpose you have described is correct

Reply:
Updated scenario description